Libraries/Components/SafeAreaView/SafeAreaView.js

```diff
@@ -35,27 +30,22 @@ let exported: React.AbstractComponent<
  * sensor housing area on iPhone X).
  */
 if (Platform.OS === 'android') {
-  exported = React.forwardRef<Props, React.ElementRef<HostComponent<mixed>>>(
-    function SafeAreaView(props, forwardedRef) {
-      const {emulateUnlessSupported, ...localProps} = props;
-      return <View {...localProps} ref={forwardedRef} />;
-    },
-  );
+  exported = React.forwardRef<
+    ViewProps,
+    React.ElementRef<HostComponent<mixed>>,
+  >(function SafeAreaView(props, forwardedRef) {
+    return <View {...props} ref={forwardedRef} />;
+  });
 } else {
   const RCTSafeAreaViewNativeComponent =
     require('./RCTSafeAreaViewNativeComponent').default;
 
-  exported = React.forwardRef<Props, React.ElementRef<HostComponent<mixed>>>(
-    function SafeAreaView(props, forwardedRef) {
-      return (
-        <RCTSafeAreaViewNativeComponent
-          emulateUnlessSupported={true}
-          {...props}
-          ref={forwardedRef}
-        />
-      );
-    },
-  );
+  exported = React.forwardRef<
+    ViewProps,
+    React.ElementRef<HostComponent<mixed>>,
+  >(function SafeAreaView(props, forwardedRef) {
+    return <RCTSafeAreaViewNativeComponent {...props} ref={forwardedRef} />;
+  });
```

Comment:
Can this be simplified to this?

```suggestion
  exported = require('./RCTSafeAreaViewNativeComponent').default;
  });
```

Reply:
Thank you for the feedback!